test/unit-tests/components/views/rooms/RoomTile-test.tsx

```diff
@@ -215,7 +216,7 @@ describe("RoomTile", () => {
             it("tracks connection state", async () => {
                 renderRoomTile();
                 screen.getByText("Video");
-                await act(() => call.start());
+                act(() => call.setConnectionState(ConnectionState.Connected));
```

Comment:
```suggestion
                await act(() => call.setConnectionState(ConnectionState.Connected));
```

Reply:
Looking at https://react.dev/reference/react/act#await-act-async-actfn, I get the impression that they expect you to make the callback async even if you don't await anything.